packages/test/test-end-to-end-tests/src/test/sharedStringEndToEndTests.spec.ts

```diff
@@ -68,4 +71,36 @@ describeFullCompat("SharedString", (getTestObjectProvider) => {
         assert.equal(
             newSharedString.getText(), text, "The new container should receive the inserted text on creation");
     });
+
+    it("marker passes on attachment directly and transitively to any referenced DDS", async () => {
+        sharedString1.insertText(0, "hello world");
```

Comment:
I don't think it matters, but why insertText here and below? Not sure what that's adding to the test.